README.md

```diff
@@ -73,4 +75,99 @@ Django-major-event-log was developed at the UNT Libraries.
 Contributors:
 
 * [Gio Gottardi](https://github.com/somexpert)
+* [Damon Kelley](https://github.com/damonkelley)
+
+
+Developing/Testing
+-------------------------
+
+#### Requirements ####
+
+- Django == 1.8
+- Python >= 2.7
+
+I suggest doing the following within a virtual environment. To do so, you will
+first need to install virtualenv with the following command:
+
+    $ [sudo] pip install virtualenv
+
+Then, you will need to set up a virtual environment by navigating to the
+location you want your environment created and running:
+
+    $ virtualenv --python=python2.7 new_virtualenv_name
+
+This will create a directory called new_virtualenv_name that is the virtual
+environment. Next, you'll need to get inside the new directory with:
+
+    $ cd new_virtualenv_name
+
+and then activate the new virtual environment:
+
+    $ source ./bin/activate
+
+(after testing, exit the virtual environment with `$ deactivate`)
+
+You will need to install Django into the virtual env to be able to test out the
+app with Django's development server:
+
+    $ pip install django==1.8
+
+Now you need to get the source files for the app by cloning the git repository:
+
+    $ git clone https://github.com/unt-libraries/django-major-event-log
+
+and navigate to within the new directory:
+
+    $ cd django-major-event-log
+
+Now, in order to be able to use the app, you must first initialize the database
+by running:
+
+    $ python manage.py migrate
+
+You should take this time to create a superuser for the app so you can create
+test events that will let you see what the site will look like with a populated
+database:
+
+    $ python manage.py createsuperuser
+
+which will guide you through creating a super-user with login credentials that
+you can use on the admin portion of the app. Now you should have a development
+environment where you can see all the source code and check out the app using
+Django's development server. To do so, simply execute the following from the
+current location (the root of the git repository):
+
+    $ python manage.py runserver
+
+(press Ctrl-c to stop the development server)
+
+You should now be able to access both the admin portion of the app at
+127.0.0.1:8000/admin/, or the public-facing side of the app at
+127.0.0.1:8000/major-event-log/.
+
+#### Running the tests ####
+
+##### Requirements #####
+
+- Django == 1.8
+- tox == 2.0
+- lxml == 3.4
+- Python == 2.7
+- Python == 3.4
+
+Simply invoke the tox test runner by running
+
+    $ tox
+
+anywhere within the app (if you are running in a virtual environment, you will
```

Comment:
I don't think we need the part in the parentheses about "if you are running" since you listed tox in requirements and virtual environment or not, you need to have it installed. I would like to see the system dependencies that are introduced by lxml listed in this section though.